pyproject.toml

```diff
@@ -1,36 +1,46 @@
-[tool.poetry]
+[project]
 name = "exasol-notebook-connector"
 version = "0.5.0"
+requires-python = ">=3.10,<4.0"
 description = "Components, tools, APIs, and configurations in order to connect Jupyter notebooks to Exasol and various other systems."
 packages = [ {include = "exasol"}, ]
-authors = [ "Christoph Kuhnke <christoph.kuhnke@exasol.com>" ]
-maintainers = [ "Christoph Kuhnke <christoph.kuhnke@exasol.com>" ]
+authors = [
+    {name="Christoph Kuhnke", email="christoph.kuhnke@exasol.com"},
+]
+maintainers= [
+    {name="Christoph Kuhnke", email="christoph.kuhnke@exasol.com"},
+]
+readme = "README.md"
+license = "MIT"
+dynamic = ["classifiers", "dependencies"]
+
+[project.urls]
+Homepage = "https://github.io/exasol/notebook-connector"
+Documentation = "https://github.io/exasol/notebook-connector"
+Source = "https://github.io/exasol/notebook-connector"
+Issues = "https://github.io/exasol/notebook-connector/issues"
+
+[tool.poetry]
+requires-poetry = ">=2.1.0"
+packages = [
```

Comment:
Sanity check: we want the package also installed in the poetry env? Without this line, you'll get an error with poetry 2.1.2 that we could alternately solve by setting package-mode = false.

Reply:
Yes, we want it also in the poetry environment